docs/setup-sensor/podman.md

```diff
@@ -0,0 +1,153 @@
+   - `AddCapability=CAP_NET_RAW`: Grants permission to capture raw network packets.
+   - `AutoUpdate=registry`: Enables automatic image updates when new versions are released.
+   - `Restart=always`: Ensures the container restarts if it stops or on system reboot.
+
+## Step 2: Start the Orb Container
+
+1. Reload systemd to recognize the quadlet. Start and enable the Orb sensor service:
+
+   ```bash
+   systemctl daemon-reload
+   systemctl start orb-sensor
+   systemctl enable orb-sensor
+   ```
+
+2. Check the status to ensure it's running:
+
+   ```bash
+   systemctl status orb-sensor
+   ```
+
+3. Verify the container is running:
+
+   ```bash
+   podman ps
+   ```
+
+   You should see the `orb-sensor` container with status `Up`. View logs with:
+
+   ```bash
+   podman logs orb-sensor
+   ```
+
+## Step 3: Link your new Orb sensor
+
+1. Once the Orb container is running, it should start broadcasting its presence on your network.
+2. Open the Orb app on your phone or personal computer (which must be on the same network).
+3. Your new Podman-based Orb sensor should be automatically detected and appear in the app, ready to be linked to your account. Follow the prompts in the app to link it.
+4. **Alternatively**, run the command `podman exec -it orb-sensor /app/orb link` from your podman host. This command generates a link to register the Orb sensor with your account. Copy and paste the link into your browser to complete registration.
+
+Congratulations! Your Orb sensor is now running as a Podman Quadlet, monitoring your network.
+
+## Troubleshooting
+
+- **Orb Not Detected:**
+  - Ensure the container is running (`podman ps`).
+  - Verify `Network=host` is set in the quadlet file. Without host networking, Orb cannot see network traffic or broadcast correctly.
+  - Check container logs (`podman logs orb-sensor`) or systemd logs (`journalctl -u orb-sensor`) for errors.
+  - Ensure your host machine's firewall is not blocking mDNS/Bonjour traffic (UDP port 5353).
+  - Make sure the device running the Orb app is on the _exact same_ network/subnet as the Podman host.
+- **Permission Errors:**
+  - Ensure you're using rootful Podman (`podman`). Rootless Podman may not support `CAP_NET_RAW` or host networking.
+  - Verify the `orb-data` volume exists (`podman volume ls`) and is accessible.
+- **Container Fails to Start:**
+  - Check systemd logs: `journalctl -u orb-sensor`.
+  - Ensure the quadlet file is correctly formatted and located in `/etc/containers/systemd/`.
+- **Registration Link Issues:**
+  - Ensure the `podman exec -it orb-sensor /app/orb link` command runs successfully and outputs a valid URL.
+  - Check container logs (`podman logs orb-sensor`) for errors related to the link command.
+- **Stopping Orb:**
+  - Run `systemctl stop orb-sensor` to stop the service.
+  - Run `systemctl disable orb-sensor` to prevent it from starting on boot.
```

Comment:
the `stop` command worked if I wanted to stop the orb, but to completely disable it you absolutely have to remove the `orb-sensor.container` file otherwise orbs will come back on reboot 😆